### DISCUSSION TITLE
Update to manifest v3

### DISCUSSION THREAD

**Discussion by professorxavi** [deathau/markdownload#331] [General]
with v2 being deprecated this will need to be updated at some point. If that something that is being looked into at all? I can take a stab at it but I want to make sure before I dive in.

**Comment by BurtHarris** [comment]
Great question.  I'm just beginning to look into this extension, but I don't know if this repro is being maintained actively.   I'd love to be wrong about that.   Anybody?

**Reply by professorxavi** [reply]
I decided to use this as a base for the specific use I had. When I get that working fully on v3, I'll port the necessary changes and open a pr if someone doesn't beat me to it.
